### PR TITLE
[W5.11][F11-A2]Colin Ng

### PR DESCRIPTION
**Pull request by niloc94**
Added Sort function, which gives a sorted list of person in the addressbook (sorted by name, alphabetically)

### CODE REVIEW

**Comment by niloc94** [comment]
Sorry for the numerous no message push commits. I was trying to do some simple adding of whitespace on IntelliJ to resolve Travis Bot test outcome but there was some issues with IntelliJ itself. Solved it by adding the spaces on text editor.

File: doc/UserGuide.md [hunk 1]
```diff
@@ -53,6 +53,11 @@ Examples:
 Shows a list of all persons in the address book.<br>
 Format: `list`
 
+### Sorting all persons : 'sort'
+Shows a list of all person in the address book sorted by name.<br>
+Format: 'sort'
+
+
```
Comment (karrui) [inline doc/UserGuide.md:60]: Extra line.

File: src/seedu/addressbook/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Sorts all person in the address book according to name
+ */
+
```
Comment (karrui) [inline src/seedu/addressbook/commands/SortCommand.java:11]: Extra line. Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 3]
```diff
@@ -81,6 +81,16 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    /**
+     * Returns a java list view with elements sorted by name
+     */
+
```
Comment (karrui) [inline src/seedu/addressbook/data/person/UniquePersonList.java:87]: Extra line.

**Review by karrui — COMMENTED** [review]
Good job attempting this LO. Please note that you were supposed to do two enhancements for this LO because you did not attempt the 1KLoC LO.

~~You will have to add another enhancement by the end of the day, please mention me when you have done so.~~

**Comment by karrui** [comment]
Note: use the imperative tone in commit messages (ex: "Add function" instead of "Added function"). Refer to: https://oss-generic.github.io/process/docs/FormatsAndConventions.html#commit-message

**Comment by karrui** [comment]
I missed your second enhancement because it's in a separate PR. Disregard this, will be reviewing it! :)

**Pull request closed without merging**